components/collective-navbar/index.js

```diff
@@ -258,7 +259,8 @@ const getDefaultCallsToActions = (collective, sections, isAdmin, isHostAdmin, Lo
     hasManageSubscriptions: isAdmin && get(features, 'RECURRING_CONTRIBUTIONS') === 'ACTIVE',
     hasDashboard: isAdmin && isFeatureAvailable(collective, 'HOST_DASHBOARD'),
     hasRequestGrant:
-      (isFund || get(settings, 'fundingRequest') === true) && expenseSubmissionAllowed(collective, LoggedInUser),
+      (get(settings, 'expenseTypes.hasGrant', get(hostSettings, 'disableGrantsByDefault', false)) || isProjectOrFund) &&
+      expenseSubmissionAllowed(collective, LoggedInUser),
```

Comment:
We're repeating this complex check in multiple places, let's extract it to a common helper (e.g. in `lib/collective.lib.js`):

```suggestion
      accountSupportsGrants(collective) && expenseSubmissionAllowed(collective, LoggedInUser),
```

Another option would be to extend the existing `expenseSubmissionAllowed`:

```suggestion
      expenseSubmissionAllowed(collective, LoggedInUser, 'GRANT'),
```

Reply:
Thanks. I've created a common function `accountSupportsGrants` in the `lib/collective.lib.js`. 👍🏽